xil_carma_cloud/cdasim_config/carma/inlanecruising_config/parameters.yaml

```diff
@@ -1,9 +1,9 @@
 # The length of the trajectory in time domain, in seconds
 trajectory_time_length: 6.0 # Trajectory length in seconds
 curve_resample_step_size: 1.0 # Curve re-sampling step size in m
-default_downsample_ratio: 18 # Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
-turn_downsample_ratio: 20 # Amount to downsample input lanelet centerline data on turns. Value corresponds to saving each nth point.
-minimum_speed: 2.2352 # Minimum allowable speed in m/s
+default_downsample_ratio: 4 # Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
+turn_downsample_ratio: 4 # Amount to downsample input lanelet centerline data on turns. Value corresponds to saving each nth point.
+minimum_speed: 0.894 # Minimum allowable speed in m/s
```

Comment:
2mph is set by workzone. So in order to allow such low speed, we need to adjust the minimum_speed to match it so as not to use higher minimum_speed and not ignore the workzone.

Reply:
Moreover, for even better performance `max_accel_multiplier` value should be `1.0` to allow full deceleration rate configured. Because it is making inlane plan only with fractional decelaration value so it is slowing down "too gradually" than expected. However, I kept it same because in this tuning, due to lci_strategic_plugin's activation distance, mostly lci_tactical_plugin is planning, so platform was not using inlane,